### DISCUSSION TITLE
fatal problem with copilot authentication from vim

### DISCUSSION THREAD

**Discussion by JFOZ1010** [community/community#34619] [Copilot Conversations]
The problem is that I wanted to install copilot on my arch linux machine, and I installed it and everything is fine but when I try to put the command ```Copilot setup`` I automatically get this error: 
![image](https://user-images.githubusercontent.com/82562585/193085706-b7b16d90-98a3-480d-887f-de4d6783a0d9.png)
I had previously been able to use copilot in vim from my macOS machine, but I do not know what happened this time with my linux distro, I hope someone can help me, thank you very much in advance.

**Comment by iamcjwilliams** [comment]
@JFOZ1010 Our copilot team is investigating this issue

**Comment by tpope** [comment]
This is fixed on the latest Copilot.vim release.
